logforward.tf

```diff
@@ -0,0 +1,72 @@
+resource "aws_cloudformation_stack" "datadog_forwarder" {
+  count = var.install_log_forwarder ? 1 : 0
+
+  name         = "${var.name}-log-forwarder"
+  capabilities = ["CAPABILITY_IAM", "CAPABILITY_NAMED_IAM", "CAPABILITY_AUTO_EXPAND"]
+  template_url = "https://datadog-cloudformation-template.s3.amazonaws.com/aws/forwarder/latest.yaml"
+
+  parameters = {
+    DdApiKeySecretArn = aws_secretsmanager_secret.datadog.arn,
+    DdSite            = var.datadog_site_name,
+    FunctionName      = "${var.name}-forwarder"
+  }
+
+  depends_on = [datadog_integration_aws.datadog]
+}
```

Comment:
why'd we switch from using an output of the main cloudformation stack to deploying a seperate cloudformation template with the forwarder?

Reply:
I don't think we changed at all? 

See..

[resource "aws_cloudformation_stack" "datadog_forwarder" {](https://github.com/rhythmictech/terraform-aws-datadog/blob/1c3dcc4b566201e932e553ce2af65d2df34020dc/main.tf#L92)

I changed on the enhanced monitoring side due to having a lot of issues with pre-commit filters both in the module itself and in projects that used the module.  Not sure if that's what you meant to ask about?